Flickr/Flickr/ImageViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  ImageViewController.swift
```

Comment:
ImageViewController is misleading name for controller. Need to be something like ImageCollectionViewController.
Need to change file name